benchmarks/converse/commbench/Makefile

```
@@ -9,8 +9,14 @@ all: commbench
 commbench: $(OBJS)
 	$(CHARMC) -o commbench $(OBJS) -language converse++
 
+HOST=$(shell hostname)
+
 test: commbench
+ifneq (,$(findstring golub,$(HOST)))
```

We should use the `:=` Makefile operator here so that the shell command is only called once at the time the variable is defined, otherwise it will be called each time the variable is used.

I also recall a prior experience where using a variable named HOST in a Makefile caused problems on certain systems where such a variable is defined in the environment, so I recommend naming it something more unique.

```suggestion
LOCALHOSTNAME:=$(shell hostname 2>/dev/null)

test: commbench
ifneq (,$(findstring golub,$(LOCALHOSTNAME)))
```

---

That makes sense, thanks @evan-charmworks.